### DISCUSSION TITLE
Invalid or unespected token js error

### DISCUSSION THREAD

**Discussion by applyACS** [filamentphp/filament#8338] [Help]
Just installed filament on an existing project laravel@10, php@8.1
All good, no errors on instalation, but when I access it I get this:
![image](https://github.com/filamentphp/filament/assets/13115651/0af15cf3-7245-4d28-9745-93e5586836dd)
Login does not work
After I login, using laravel route, this is how the dashboard looks, with the same js error:
![image](https://github.com/filamentphp/filament/assets/13115651/b96eb50b-cc87-4a7d-bb51-04365fff6a0a)
I tried writing a Users resource, but still the dashboard does not work
And any save is not working
![image](https://github.com/filamentphp/filament/assets/13115651/f7a431c3-5be5-46b4-b268-c956924b4b83)
What am I doing wrong?

This is my composer
`
{
    "keywords": ["framework", "laravel"],
    "license": "MIT",
    "type": "project",
    "require": {
        "php": ">=8.1",
        "barryvdh/laravel-snappy": "^1.0",
        "doctrine/dbal": "^3.0",
        "doctrine/inflector": "^2.0",
        "filament/filament": "^3.0-stable",
        "giggsey/libphonenumber-for-php-lite": "^8.13",
        "google/apiclient": "^2.12.1",
        "guzzlehttp/guzzle": "^7.0",
        "guzzlehttp/psr7": "^2.0",
        "intervention/image": "2.*",
        "laracasts/flash": "^3.0",
        "laravel/framework": "^10.0",
        "laravel/horizon": "^5.7",
        "laravel/ui": "^4.0",
        "laravelcollective/html": "^6.2",
        "livewire/livewire": "^3.0",
        "lodge/postcode-lookup": "dev-master",
        "maatwebsite/excel": "^3.1",
        "maennchen/zipstream-php": "^2.1",
        "messagebird/php-rest-api": "1.0",
        "owen-it/laravel-auditing": "^13.0",
        "php-http/curl-client": "^2.2",
        "sentry/sentry-laravel": "^3.7",
        "spatie/laravel-backup": "^8.0",
        "spatie/laravel-permission": "^5.0"
    },
    "require-dev": {
        "barryvdh/laravel-debugbar": "^3.1",
        "fzaninotto/faker": "~1.4",
        "laravel/tinker": "^2.0",
        "mockery/mockery": "0.9.*",
        "phpunit/phpunit": "^9.5",
        "squizlabs/php_codesniffer": "^3.4",
        "spatie/laravel-ignition": "^2.0",
        "symfony/css-selector": "~3.0",
        "symfony/dom-crawler": "~3.0"
    },
    "autoload": {
        "files": [
            "app/helpers.php"
        ],
        "classmap": [
            "database"
        ],
        "psr-4": {
            "App\\": "app/"
        }
    },
    "autoload-dev": {
        "psr-4": {
            "Tests\\": "tests/"
        }
    },
    "scripts": {
        "post-autoload-dump": [
            "Illuminate\\Foundation\\ComposerScripts::postAutoloadDump",
            "@php artisan package:discover --ansi",
            "@php artisan filament:upgrade"
        ],
        "post-root-package-install": [
            "php -r \"copy('.env.example', '.env');\""
        ],
        "post-create-project-cmd": [
            "php artisan key:generate"
        ]
    },
    "config": {
        "preferred-install": "dist",
        "sort-packages": true,
        "optimize-autoloader": true,
        "allow-plugins": {
            "php-http/discovery": true
        }
    },
    "minimum-stability": "stable",
    "prefer-stable": true
}
`

**Comment by leandrocfe** [comment]
`php artisan filament:upgrade` ?
clear cache?

**Reply by applyACS** [reply]
tried all that. Same result
